modules/custom/openy_campaign/src/Plugin/Block/CampaignRegisterBlock.php

```diff
@@ -136,10 +136,10 @@ public function build() {
       $activeRegistration = FALSE;
     }
 
-    $utcCampaignStart = $localeCampaignStart->getDate()->format(DATETIME_DATETIME_STORAGE_FORMAT);
-    $utcCampaignEnd = $localeCampaignEnd->getDate()->format(DATETIME_DATETIME_STORAGE_FORMAT);
-    $utcCampaignRegStart = $localeRegistrationStart->getDate()->format(DATETIME_DATETIME_STORAGE_FORMAT);
-    $utcCampaignRegEnd = $localeRegistrationEnd->getDate()->format(DATETIME_DATETIME_STORAGE_FORMAT);
+    $utcCampaignStart = $localeCampaignStart->getDate()->format(\Drupal\datetime\Plugin\Field\FieldType\DateTimeItemInterface::DATETIME_STORAGE_FORMAT);
```

Comment:
Namespaced classes/interfaces/traits should be referenced with use statements

Reply:
fixed https://github.com/ymcatwincities/openy/pull/2174/commits/839562bfa412cf71c9862ca9d9ce54b72f92ff06